### ISSUE TITLE
ModalPopup - NotFound javascript error on IE10

### ISSUE THREAD

**Issue by savornicesei** [DevExpress/AjaxControlToolkit#17]
Hello,

I have a WebForm app. using .NET 4.5 and AjaxControlToolkit 16.1  and in IE10 I get a first NotFound javascript error in ModalPopup.js at line 131:
```javascript
this._foregroundElement.parentNode.removeChild(this._backgroundElement);
```
and then an "Unable to set property 'actControlType' of undefined or null reference' at line 180:
```javascript
this._backgroundElement.dataset.actControlType = "modalPopupBackground";
```
It works fine in IE Edge, Chrome and Firefox.
Not sure if it counts but the ModalPopupExtender that fails is declared in a TabPanel (complete hierarchy is asp:Content > ajaxToolkit:TabContainer > ajaxToolkit:TabPanel > ContentTemplate > ajaxToolkit:ModalPopupExtender)

**Comment by savornicesei** [comment]
As a workaround for the current 16.1 version of AjaxControlToolkit use [https://gist.github.com/brettz9/4093766](url)

**Comment by dparks2** [comment]
@savornicesei , that link does not work but I love the 404 error.

**Comment by cluehawk** [comment]
Confirmed in IE11 - Unable to set property 'actControlType' is a problem here too.
Rears is head when setting the BackgroundCssClass.
Worked fine in AJAXControlToolKit 15.1.4.0

**Comment by savornicesei** [comment]
@dparks2 : Thanks. I fixed the link.

**Comment by motleydrew** [comment]
I'm seeing this as closed, but I do not see the resolution. I attempted to use the html5-dataset.js workaround for IE11 and ATC16.1, but it doesn't work. I'm still getting the, **"Unable to set property 'actControlType' of undefined or null reference"**, error. 

One thing to note; it works in IE11 (even without the html5-dataset.js) if I launch it from Visual Studio on my local machine. It fails when I deploy to the web server.

**Comment by AlekseyMartynov** [comment]
@motleydrew 

> I'm seeing this as closed, but I do not see the resolution.

We use the ["Closing issues via commit messages"](https://help.github.com/articles/closing-issues-via-commit-messages/) practice. The "Closed" state means that a fix has been merged into the master branch and will be available in the next release of Toolkit. Before that, it's possible to clone or download the Git repository and compile a custom build from source code.

**Comment by manuelra** [comment]
@savornicesei 
What's the right recipe for using html5-dataset.js? (I tried adding it on script tag to my master page but that did not suffice)

@AlekseyMartynov 
I cloned + compiled the repository. And then noticed that the assembly version on this repository is still 15.1.2. Is this correct? does this code match the one for v 16.1?

Thank you

**Comment by AlekseyMartynov** [comment]
@manuelra 
Thanks for bringing our attention. We'll fix that in the context of PR #117

**Comment by Tagmar2** [comment]
html5-dataset.js works! 

My solution:

` 
If Request.Browser.Browser = "IE" Then
            Dim Vesao = Request.Browser.Version

            If Vesao = "10.0" OrElse Vesao = "9.0" Then
                ToolkitScriptManager1.Scripts.Add(New ScriptReference("~/Scripts/html5-dataset.js"))
            End If
  End If`
